aioax25/kiss.py

```diff
@@ -12,6 +12,9 @@
 from binascii import b2a_hex
 import time
 import logging
+import asyncio
+import serial_asyncio
+import types
```

Comment:
Is `types` actually used anywhere?  I realise this is harmless, but I usually prefer to not import things unless it's actually used somewhere in the file.